content/courses/intro-to-spark/chapters/05_Proof_Of_Functional_Correctness.rst

```diff
@@ -687,6 +701,9 @@ isn't part of the functional behavior of the subprogram. Finally, it can
 help GNATprove by abstracting away assertions that would otherwise make its
 job more complex.
 
+.. todo::
+    What is a Proof Context? I don't think this is defined anywhere.
```

Comment:
Minor editorial comment: an empty line is missing after the todo.
```suggestion
.. todo::

    What is a Proof Context? I don't think this is defined anywhere.
```

Reply:
the "proof context" is the logical formula sent to the prover. Maybe add after "proof context" the explanation (i.e. the logical formula sent to automatic provers)